packages/rocketchat-ui-admin/client/adminFlex.html

```diff
@@ -1,13 +1,15 @@
 <template name="adminFlex">
 	<aside class="sidebar-light" role="navigation">
-		<header class="sidebar__header">
-			{{> accountBox lighten=true modifier="--admin"}}
-			<button class="sidebar-flex__back-button" data-action="back">
-				{{> icon block="sidebar-flex__back-icon" icon="back"}}
-				<span>{{_ "Back_to_chat"}}</span>
-			</button>
-		</header>
-		<div class="rooms-list" aria-label="{{_ "Administration"}}">
+		{{#unless embeddedVersion}}
+			<header class="sidebar__header">
+				{{> accountBox lighten=true modifier="--admin"}}
+				<button class="sidebar-flex__back-button" data-action="back">
+					{{> icon block="sidebar-flex__back-icon" icon="back"}}
+					<span>{{_ "Back_to_chat"}}</span>
```

Comment:
Change "Back_to_chat" to just "Back"

Reply:
Why?

Reply:
Less is More

Reply:
ok, it's easy to maintain translations :)